lib/transloadit/exceptions.rb

```diff
@@ -0,0 +1,21 @@
+require 'transloadit'
+
+require 'rest-client'
+
+#
+# Transloadit custom Exception classes
+#
+module Transloadit::Exception
+
+  #
+  # Exception raised when Rate limit error response is returned from the API.
+  # See {Rate Limiting}[https://transloadit.com/docs/api-docs/#rate-limiting]
+  #
+  class RateLimitReached < RestClient::RequestEntityTooLarge
```

Comment:
Right! The transloadit api happens to return a 413 for ratelimits as seen [here](https://transloadit.com/docs/api-docs/#rate-limiting)